dashboard/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180601174113) do
+ActiveRecord::Schema.define(version: 20180531011848) do
```

Comment:
Is this right?

Reply:
No, I also just noticed that. I needed to run an additional migration. Updating...